tedana/tests/test_component_selector.py

```diff
@@ -36,7 +36,6 @@ def dicts_to_test(treechoice):
         "extra_opt_param": A tree with an undefined optional parameter for a decision node function
         "missing_req_param": A missing required param in a decision node function
         "missing_function": An undefined decision node function
```

Comment:
```suggestion
        "missing_function": An undefined decision node function
        "missing_key": A dict missing one of the required keys (report)
```

We want to test whether the dict is missing one of the required keys! We just need to change it to another key.

Reply:
The check seems to be specific to `refs`, unless I was misreading the code.

Reply:
Ah, I just saw your original comment. Never mind.